src/features/settings/screens/UsersScreen/UsersScreen.tsx

```diff
@@ -74,8 +74,7 @@ const UsersScreen = () => {
   };
 
   return (
-    <>
-      <ScreenTitleView title={t("settingsUsers.title")} />
+    <ContentLayout header={{ title: t("settingsUsers.title") }}>
```

Comment:
can you move `<UsersFilter />` up to `<ContentLayout filters={<UsersFilter />}` so the ContentLayout is used as intended? I forgot to add that 😊

Reply:
this is added now :)